runtime/near-vm-runner/src/tests/cache.rs

```diff
@@ -71,21 +75,41 @@ fn make_cached_contract_call_vm(
     let promise_results = vec![];
     context.prepaid_gas = prepaid_gas;
     let code = ContractCode::new(code.to_vec(), None);
-    let runtime = vm_kind.runtime().expect("runtime has not been compiled");
-
+    let runtime = vm_kind.runtime(config).expect("runtime has not been compiled");
     runtime.run(
         &code,
         method_name,
         &mut fake_external,
         context.clone(),
-        &config,
         &fees,
         &promise_results,
         LATEST_PROTOCOL_VERSION,
         Some(cache),
     )
 }
 
+#[test]
+fn test_wasmer2_artifact_output_stability() {
+    // If this test has failed, you want to adjust the necessary constants so that `cache::vm_hash`
+    // changes (and the only then the hashes here).
```

Comment:
Something's up with grammar here? "the only then the hashes"? love the alliteration though